projects/1.20/assets/enderite-cj-edition/enderite/lang/zh_cn.json

```
@@ -0,0 +1,113 @@
+  "tooltip.enderite.enderite_armor_set_bonus_over_value": "生效时护甲耐久需大于",
+  "tooltip.enderite.enderite_helmet": "直视末影人将不再激怒它",
+  "tooltip.enderite.enderite_elytra_chestplate": "你能飞了！",
+  "tooltip.enderite.enderite_leggings_press_shift": "按下潜行键以解除飘浮效果",
+  "tooltip.enderite.enderite_leggings_op_mode": "移除飘浮效果",
+  "tooltip.enderite.enderite_boots": "让你在末地石上移动得更快",
+  "tooltip.enderite.enderite_boots_op_mode": "允许你在末地更快地移动",
+  "tooltip.enderite.enderite_boots_configurable": "你可以在'data/forge/tags/blocks/enderite_boots_walks_faster_on_blocks.json'中配置获得速度的方块",
+
+  "advancement.end.enderite.charge_end_respawn_anchor.title": "好奇心害死锚",
+  "advancement.end.enderite.charge_end_respawn_anchor.description": "将末地重生锚充能至最大",
+  "advancement.end.enderite.explore_all_end_biomes.title": "冷门景点",
+  "advancement.end.enderite.explore_all_end_biomes.description": "探索所有末地生物群落",
+  "advancement.end.enderite.obtain_ender_crystal_shard.title": "末影水晶碎片？",
+  "advancement.end.enderite.obtain_ender_crystal_shard.description": "找到一个末地晶洞！",
+  "advancement.end.enderite.obtain_enderite_ingot.title": "这东西好吗？",
+  "advancement.end.enderite.obtain_enderite_ingot.description": "获得末影合金锭",
+  "advancement.end.enderite.obtain_enderite_upgrade_smithing_template.title": "又一项升级？",
+  "advancement.end.enderite.obtain_enderite_upgrade_smithing_template.description": "获得一个末影合金升级锻造模板",
+  "advancement.end.enderite.destroyer_of_the_end.title": "末地毁灭者！",
+  "advancement.end.enderite.destroyer_of_the_end.description": "获得末影合金镐",
+  "advancement.end.enderite.obtain_enderite_armor.title": "末影裹身",
+  "advancement.end.enderite.obtain_enderite_armor.description": "获得一套完整的末影合金盔甲",
+  "advancement.end.enderite.obtain_enderite_elytra_chestplate.title": "完美无缺的盔甲",
+  "advancement.end.enderite.obtain_enderite_elytra_chestplate.description": "获得末影合金鞘翅胸甲",
+
+  "advancement.husbandry.enderite.obtain_enderite_hoe.title": "究极奉献",
+  "advancement.husbandry.enderite.obtain_enderite_hoe.description": "获得末影合金锄",
+
+  "item.enderite.enderite_upgrade_smithing_template": "末影合金升级",
+  "upgrade.enderite.enderite_upgrade": "末影合金升级",
+  "item.enderite.smithing_template.enderite_upgrade.additions_slot_description": "在这里放一个末影合金锭或一个鞘翅",
+  "item.enderite.smithing_template.enderite_upgrade.additions_slot_description_without_elytra": "在这里放一个末影合金锭",
+  "item.enderite.smithing_template.enderite_upgrade.applies_to": "下界合金装备，末影合金胸甲",
+  "item.enderite.smithing_template.enderite_upgrade.applies_to_without_elytra": "下界合金装备",
+  "item.enderite.smithing_template.enderite_upgrade.base_slot_description": "在这里放一件下界合金盔甲、武器或工具。或者把末影合金胸甲放在这里",
+  "item.enderite.smithing_template.enderite_upgrade.base_slot_description_without_elytra": "在这里放一件下界合金盔甲、武器或工具",
+  "item.enderite.smithing_template.enderite_upgrade.ingredients": "末影合金锭，鞘翅",
+  "item.enderite.smithing_template.enderite_upgrade.ingredients_without_elytra": "末影合金锭",
+  "trim_material.enderite.enderite": "末影合金材料（类似飘浮I）",
```

```suggestion
  "trim_material.enderite.enderite": "末影合金材料（效果类似虚空飘浮I）",
```
漏翻
可以不译出`效果`，但飘浮会和原版levitation撞